org-formation/075-security-hub/security-hub-suppress-infra.yaml

```diff
@@ -403,6 +403,8 @@ Resources:
                 - prefix: 'arn:aws:s3:::bootstrap-awss3cloudformationbucket-'
                 - prefix: 'arn:aws:s3:::essentials-awss3lambdaartifactsbucket-'
                 - prefix: 'arn:aws:s3:::cf-templates-'
+                - prefix: 'arn:aws:s3:::cloudformationmanageduploadinfrast-'
+                - prefix: 'arn:aws:s3:::cost-and-usage-reports.sagebase.orggi'
```

Comment:
there's a small typo in this name: `.orggi` should be `.org`

Reply:
fixed